### PR TITLE
landing page of E-commerce -

### PR DESCRIPTION
**Pull request by ajit-sharmaGH**
-different categories of collection products
-different services features

### CODE REVIEW

**Review by Kothalkarpriya — APPROVED** [review]
Looks good to me

File: style.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    height: 2rem;
+    width: 30vw;
+    outline: none;
+    border: 1px solid var(--dark-primary);
+    border-radius: 5px;
+    padding: 1px 5px;
+    box-shadow: 1px 1px 2px 2px var(--dark);
+    font-family: monospace;
+    font-weight: 600;
+    color: #333;
```
Comment (ozairkhan2810) [inline style.css:29]: use color variable

**Review by ozairkhan2810 — APPROVED** [review]
looks good, 
just one suggestion and then can merge

Also add image or gif or preview link while raising PR

**Comment by ajit-sharmaGH** [comment]
image

**Comment by ajit-sharmaGH** [comment]
image

**Comment by ajit-sharmaGH** [comment]
> looks good, 
> just one suggestion and then can merge
> 
> Also add image or gif or preview link while raising PR

Thank you very much for the suggestion! I was not aware of that!

**Pull request merged**